Libraries/CMSIS/Device/Maxim/MAX32570/Include/adc9_regs.h

```diff
@@ -70,322 +77,265 @@ extern "C" {
 /* **** Definitions **** */
 
 /**
- * @ingroup     adc
- * @defgroup    adc_registers ADC_Registers
- * @brief       Registers, Bit Masks and Bit Positions for the MSRADC Peripheral Module.
- * @description Analog to Digital Converter
+ * @ingroup     adc9
+ * @defgroup    adc9_registers ADC9_Registers
+ * @brief       Registers, Bit Masks and Bit Positions for the ADC9 Peripheral Module.
+ * @details     Magnetic Strip Reader - 9 bit ADC
  */
 
 /**
- * @ingroup adc_registers
- * Structure type to access the MSRADC Registers.
+ * @ingroup adc9_registers
+ * Structure type to access the ADC9 Registers.
  */
 typedef struct {
-    __IO uint32_t cfg;                  /**< <tt>\b 0x00:<\tt> MSRADC CFG Register */
-    __IO uint32_t cmd;                  /**< <tt>\b 0x04:<\tt> MSRADC CMD Register */
-    __IO uint32_t fifo;                 /**< <tt>\b 0x08:<\tt> MSRADC FIFO Register */
-    __IO uint32_t intr;                  /**< <tt>\b 0x0C:<\tt> MSRADC INTR Register */
-    __IO uint32_t stat;                 /**< <tt>\b 0x10:<\tt> MSRADC STAT Register */
-} mxc_msradc_regs_t;
-
-/* Register offsets for module MSRADC */
+    __IO uint32_t cfg;                  /**< <tt>\b 0x0000:</tt> ADC9 CFG Register */
+    __IO uint32_t cmd;                  /**< <tt>\b 0x0004:</tt> ADC9 CMD Register */
+    __IO uint32_t fifo;                 /**< <tt>\b 0x0008:</tt> ADC9 FIFO Register */
+    __IO uint32_t intr;                 /**< <tt>\b 0x000C:</tt> ADC9 INTR Register */
+    __IO uint32_t stat;                 /**< <tt>\b 0x0010:</tt> ADC9 STAT Register */
+} mxc_adc9_regs_t;
```

Comment:
The MSRADC was the new official name chosen for our latest parts. The ME13 used adc9 though. To prevent breaking any code dependent on adc9_regs.h and to organize the SVD files, I left the peripheral name as adc9 in the SVD file, but I changed the name of the SVD file to MSRADC.